src/extensions/replay/sessionrecording-utils.ts

```diff
@@ -165,27 +166,154 @@ export function truncateLargeConsoleLogs(_event: eventWithTime) {
 
 export const SEVEN_MEGABYTES = 1024 * 1024 * 7 * 0.9 // ~7mb (with some wiggle room)
 
-// recursively splits large buffers into smaller ones
+function sliceList(list: any[], sizeLimit: number): any[][] {
+    const size = estimateSize(list)
```

Comment:
We compute the size in this function but then always need it again in the calling functions. Two examples:
https://github.com/PostHog/posthog-js/pull/1307/files#diff-deb180b570d4a9e728cd83fd418e54a748f69492d07c9809924a3821ca4de4d5R187
https://github.com/PostHog/posthog-js/pull/1307/files#diff-deb180b570d4a9e728cd83fd418e54a748f69492d07c9809924a3821ca4de4d5R243

If estimating the size is an expensive operation perhaps we should look to return the value so it doesn't need to be recomputed

Reply:
i think this has either changed enough the comment doesn't apply any more, or I don't understand the comment 🙈 🤣

Reply:
It's not a biggie, just noticed that we estimated the size while slicing the list but they re-estimate in https://github.com/PostHog/posthog-js/pull/1307/files#diff-deb180b570d4a9e728cd83fd418e54a748f69492d07c9809924a3821ca4de4d5R184-R186. Could possibly just return the already estimated size